### ISSUE TITLE
Please Embed linuxx64_odbc_cli.tar.gz into the pypi package.

### ISSUE THREAD

**Issue by WaylonWalker** [ibmdb/python-ibmdb#640]
Currently, public.dhe.ibm.com is down.  Since the ibm_db downloads the driver from there during install I cannot install ibm_db, my deployment pipelines are broken and I cannot ship my product.

This is the link to the file

https://public.dhe.ibm.com/ibmdl/export/pub/software/data/db2/drivers/odbc_cli/linuxx64_odbc_cli.tar.gz

**Comment by amukherjee28** [comment]
Hi @WaylonWalker 

This should be working fine now. Can you please confirm.

**Comment by amukherjee28** [comment]
closing this issue as this was temporary downtime with the download site for clidriver packages.

**Comment by nicpayne713** [comment]
This issue has arisen again in my CI:
```
Collecting ibm-db
  Downloading ibm_db-3.1.0.tar.gz (797 kB)
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: still running...
  Getting requirements to build wheel: finished with status 'error'
  ERROR: Command errored out with exit status 1:
   command: /opt/hostedtoolcache/Python/3.8.12/x64/bin/python /opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py get_requires_for_build_wheel /tmp/tmpaxeudlsu
       cwd: /tmp/pip-install-2c06zk_g/ibm-db_6131c0adfc784e9eb59d56029a8d5100
  Complete output (30 lines):
  Detected 64-bit Python
  Downloading https://public.dhe.ibm.com/ibmdl/export/pub/software/data/db2/drivers/odbc_cli/linuxx64_odbc_cli.tar.gz
  Traceback (most recent call last):
    File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 363, in <module>
      main()
    File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 345, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 130, in get_requires_for_build_wheel
      return hook(config_settings)
    File "/tmp/pip-build-env-__vio3ps/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 162, in get_requires_for_build_wheel
      return self._get_build_requires(
    File "/tmp/pip-build-env-__vio3ps/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 143, in _get_build_requires
      self.run_setup()
    File "/tmp/pip-build-env-__vio3ps/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 158, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 240, in <module>
      file_stream = BytesIO(request.urlopen(url, context=context).read())
    File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/urllib/request.py", line 222, in urlopen
      return opener.open(url, data, timeout)
    File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/urllib/request.py", line 531, in open
      response = meth(req, response)
    File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/urllib/request.py", line 640, in http_response
      response = self.parent.error(
    File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/urllib/request.py", line 569, in error
      return self._call_chain(*args)
    File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/urllib/request.py", line 502, in _call_chain
      result = func(*args)
    File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/urllib/request.py", line 649, in http_error_default
      raise HTTPError(req.full_url, code, msg, hdrs, fp)
  urllib.error.HTTPError: HTTP Error 500: Internal Server Error
```

Can the tar files be embedded into the python package directly?

**Comment by WaylonWalker** [comment]
My CI is down again today because of this issue, can you please embed this directly in the wheel or host this on a reliable static host?

**Comment by WaylonWalker** [comment]
Fixed in #771.  Thank You so much @Earammak !!!!!  Like seriously thank you!
